test/test_unspec.py

```diff
@@ -133,6 +133,7 @@ def fn(x):
         res2 = opt_fn(x)
         self.assertTrue(same(res1, res2))
 
+    @patch.object(torchdynamo.config, "fake_tensor_propagation", False)
```

Comment:
I'm curious why this failed? This is one of the test case copied from Pytorch CI, where fake tensor is enabled IIRC.

Reply:
```
>>> x = torch.tensor(0)
>>> torch.randint(x, x, x)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: randint() received an invalid combination of arguments - got (Tensor, Tensor, Tensor), but expected one of:
 * (int high, tuple of ints size, *, torch.Generator generator, Tensor out, torch.dtype dtype, torch.layout layout, torch.device device, bool pin_memory, bool requires_grad)
 * (int high, tuple of ints size, *, Tensor out, torch.dtype dtype, torch.layout layout, torch.device device, bool pin_memory, bool requires_grad)
 * (int low, int high, tuple of ints size, *, torch.Generator generator, Tensor out, torch.dtype dtype, torch.layout layout, torch.device device, bool pin_memory, bool requires_grad)
 * (int low, int high, tuple of ints size, *, Tensor out, torch.dtype dtype, torch.layout layout, torch.device device, bool pin_memory, bool requires_grad)
 ```
 
 the int->torch.tensor(0) conversion causes an error